src/components/Icon/Icon.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { render, act } from '../../../test-utils';
+
+import {Icon} from './Icon';
+
+describe('Icon', () => {
+
+	it('should render from svg', () => {
+		const wrapper = render(<Icon name="svg-dd" />);
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
Shouldn't be better to match testing library usage, instead of enzyme kind of api?
```suggestion
		const { container } = render(<Icon name="svg-dd" />);
		expect(container.firstChild).toMatchSnapshot();
```